src/main.js

```diff
@@ -41,7 +41,8 @@ const load = loader(Object.assign({
     requires: ['cfg', 'monitor'],
     setup: ({cfg, monitor}) => new pulse.Client({
       monitor,
-      ...cfg.pulse,
+      namespace: cfg.pulse.namespace,
+      credentials: pulse.pulseCredentials(cfg.pulse),
```

Comment:
I think you're sending in the namespace as well, is that ok?

Reply:
Yeah, it's just an extra property.  I guess it's sort of cheating, but I got away with it :)